### PR TITLE
Enable analytics for Huawei devices

### PR DESCRIPTION
**Pull request by felipeerias**
Enable `HiAnalytics` in `PlatformActivity`.

Rename `HVR_ML_API_KEY` to `HVR_API_KEY`.

### CODE REVIEW

**Review by svillar — CHANGES_REQUESTED** [review]
I am not sure the rename pays off as it'll force developers to update their userProperties files. Is there any strong reason to do that?

If the answer is yes, then I'd ask you to split this change in two different commits, one for the rename and some other for the actual fix.

File: app/src/hvr/java/com/igalia/wolvic/PlatformActivity.java [hunk 1]
```diff
@@ -87,6 +90,11 @@ protected void onCreate(Bundle savedInstanceState) {
         mLocationManager = new HVRLocationManager(this);
         PermissionDelegate.sPlatformLocationOverride = session -> mLocationManager.start(session);
 
+        // Enable analytics
+        HiAnalyticsTools.enableLog();
```
[the remark this answers is not in the document]
Comment (felipeerias) [reply app/src/hvr/java/com/igalia/wolvic/PlatformActivity.java:94]: I have changed it so it is only enabled in debug builds.

**Comment by felipeerias** [comment]
@svillar It is a small thing, but I think that renaming `HVR_ML_API_KEY` to `HVR_API_KEY` would make sense. I will open a separate PR and we talk about it there.

**Review by svillar — APPROVED** [review]
Nice!. Thanks

**Comment by svillar** [comment]
> @svillar It is a small thing, but I think that renaming `HVR_ML_API_KEY` to `HVR_API_KEY` would make sense. I will open a separate PR and we talk about it there.

Sure, I'm fine with renamings but I prefer specific PRs for them, that helps with bisects, blames etc...

**Pull request merged**